### ISSUE TITLE
Consider integrating the AWS Lambda commit into gitlab

### ISSUE THREAD

**Issue by vortexing** [FredHutch/wiki#770]
Right now last modified dates are continuously updated vi an automated commit after a AWS Lamba process.  We should consider sometime integrating that into the newer GitLab build process so that we have fewer moving parts.

**Comment by dtenenba** [comment]
Good idea, especially as we need to get rid of the AWS account where that lambda function runs. We should bear in mind though that this process will still create new commits.

**Comment by laderast** [comment]
@dtenenba is this already done? Can we close?

**Comment by dtenenba** [comment]
Not done. Thanks for the reminder.
